lib/fika/erl_translate.ex

```diff
@@ -40,6 +40,11 @@ defmodule Fika.ErlTranslate do
     {:op, line, bin_op, translate_exp(arg1), translate_exp(arg2)}
   end
 
+  defp translate_exp({:call, {un_op, {line, _, _}}, [arg1], _module})
+       when un_op in [:+, :-] do
+    {:op, line, un_op, translate_exp(arg1)}
+  end
```

Comment:
Minor nitpick - we can just call it `arg` instead of `arg1` because it's the only arg